src/styles/bramble_overrides.css~

```diff
@@ -0,0 +1,74 @@
+/* Menu item goes to next line due to float:right issue on Firefox */
```

Comment:
Is this a backup file? If so, this should probably be removed from the PR (see @humphd's comment on how to do that for the config file already)